br/pkg/task/common.go

```diff
@@ -355,7 +355,7 @@ func (cfg *Config) parseCipherInfo(flags *pflag.FlagSet) error {
 	}
 
 	if !checkCipherKeyMatch(&cfg.CipherInfo) {
-		return errors.Annotate(err, "Cipher type and key not match")
+		return errors.Annotatef(berrors.ErrInvalidArgument, "crypter method and key not match")
```

Comment:
```suggestion
		return errors.Annotate(berrors.ErrInvalidArgument, "crypter method and key length not match")
```

(or maybe keep using `Annotatef` but report the expected and actual length)